>From 89d46e3cfbd8795f2635c30b809079903a12008f Mon Sep 17 00:00:00 2001
From: Pino Toscano <toscano.pino@tiscali.it>
Date: Wed, 27 Jul 2016 22:24:17 +0200
Subject: [PATCH] Build the voxware server on Hurd

While Hurd has no audio yet, at least this brings nas on Hurd on par
with what is done currently on Linux and *BSD.
---
 server/Imakefile          | 4 ++--
 server/dia/auservertype.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/server/Imakefile b/server/Imakefile
index 72587d0..6ab761e 100644
--- a/server/Imakefile
+++ b/server/Imakefile
@@ -96,10 +96,10 @@ ALL1 = nasd
 #endif
 
 
-#if defined(i386SVR4Architecture) || defined(__FreeBSD_kernel__) || defined(__FreeBSD__) || defined(FreeBSDArchitecture) || defined(LinuxArchitecture) || defined(i386BsdArchitecture) || defined(USLArchitecture) || defined(cygwinArchitecture) || defined(__DragonFly__)
+#if defined(i386SVR4Architecture) || defined(__FreeBSD_kernel__) || defined(__FreeBSD__) || defined(FreeBSDArchitecture) || defined(LinuxArchitecture) || defined(i386BsdArchitecture) || defined(USLArchitecture) || defined(cygwinArchitecture) || defined(__DragonFly__) || defined(GNUArchitecture)
 # define CanBuildAuServer	YES
 
-#if !defined(GNUMachArchitecture)
+#if !defined(GNUMachArchitecture) || defined(GNUArchitecture)
 # if defined(cygwinArchitecture)
 ALL1 = nasd.exe
 # else
diff --git a/server/dia/auservertype.h b/server/dia/auservertype.h
index 9e0f392..cb5c12f 100644
--- a/server/dia/auservertype.h
+++ b/server/dia/auservertype.h
@@ -14,7 +14,7 @@
 # define SGI_SERVER
 #endif /* sgi */
 
-#if defined(__DragonFly__) || defined(__FreeBSD_kernel__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(linux) || (defined(SVR4) && (defined(SYSV386) || defined(i386))) || defined(__CYGWIN__)
+#if defined(__DragonFly__) || defined(__FreeBSD_kernel__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(linux) || (defined(SVR4) && (defined(SYSV386) || defined(i386))) || defined(__CYGWIN__) || defined(__GNU__)
 # define VOXWARE_SERVER
 #endif /* voxware */
 
-- 
2.8.1

